integration/src/main/java/scripts/testscripts/PrivateControlledGcsBucketLifecycle.java

```diff
@@ -200,7 +200,7 @@ public void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspace
     assertThat(
         ex.getMessage(),
         containsString(
-            "PrivateResourceUser can only be specified by applications for private resources"));
+            "Request could not be parsed or was invalid: MethodArgumentNotValidException. Ensure that all types are correct and that enums have valid values."));
```

Comment:
nit: checking for the whole error message is pretty brittle. Could this just assert something like `containsString("MethodArgumentNotValidException")`?

Reply:
Good point, will do that.